resources/sql/queries.sql

```diff
@@ -125,6 +126,23 @@ INSERT INTO issues (repo_id, issue_id, issue_number, title)
                    FROM issues
                    WHERE repo_id = :repo_id AND issue_id = :issue_id);
 
+-- :name remove-issue! :<! :1
+-- :doc removes issue
+WITH archived AS (
+  INSERT INTO archive(type,data)
```

Comment:
How about describing this in clojure? I think that it is better to keep persistence as simple as possible and to have all of the logic in the the application code.

Reply:
I think it's simple enough to stay in a query, plus one roundtrip less to DB is a good thing.